content/en/docs/project-user-guide/application-workloads/routes.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Routes"
+keywords: "KubeSphere, Kubernetes, Route, Ingress"
+description: "How to create a Route in KubeSphere"
+weight: 10270
+---
+
+This document describes how to create, use, and edit a Route on KubeSphere.
+
+A Route on KubeSphere is the same as an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/#what-is-ingress) on Kubernetes. You can use a Route and a single IP address to aggregate and expose multiple services.
+
+## Prerequisites
+
+- You need to create a workspace, a project and an account (for example, `project-admin`). The role of the account in the project must be `admin`. For more information, see [Create Workspaces, Projects, Accounts and Roles](/docs/quick-start/create-workspace-and-project/).
+- If the Route is to be accessed in HTTPS mode, you need to [create a Secret](/docs/project-user-guide/configuration/secrets/) that contains the `tls.crt` (TLS certificate) and `tls.key` (TLS private key) keys used for encryption.
+- You need to [create at least one service](/docs/project-user-guide/application-workloads/services/). This document uses a demo service as an example, which returns the pod name to external requests.
+
+## Configure the Route Access Method
+
+1. Log in to the KubeSphere web console as `project-admin` and go to your project.
+
+2. Choose **Project Settings** > **Advanced Settings** on the left navigation bar and click **Set Gateway** on the right.
```

Comment:
Choose **Project Settings** > **Advanced Settings** on the left navigation bar
->
Choose **Advanced Settings** in **Project Settings** on the left navigation bar